webapp/src/containers/SettingsPage/components/SettingsTabSecurity/index.tsx

```diff
@@ -53,36 +50,47 @@ export const MaxTimeout = {
   value: TimeoutLockEnum.MAX_TIME,
 };
 
-const SettingsTabSecurity: React.FunctionComponent = () => {
-  const dispatch = useDispatch();
-  const { lockTimeoutList, defaultLockTimeout } = useSelector(
-    (state: RootState) => state.settings
-  );
-  const { myMasternodes } = useSelector(
-    (state: RootState) => state.masterNodes
-  );
+interface SettingsTabSecurityProps {
```

Comment:
same here

Reply:
I have been using connect, mapStateToProps and mapDispatchToProps throughout the project thats why I am getting all the states and actions from props